app/src/main/res/layout/ticket_settings_fragment.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <android.support.design.widget.CoordinatorLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/color_top_surface">
+
+        <View
+            android:background="@color/grey_400"
+            android:layout_height="1dp"
+            android:layout_width="match_parent" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:id="@+id/restrict_all"
+                android:layout_height="wrap_content"
+                android:layout_width="match_parent"
+                android:orientation="horizontal"
+                android:padding="@dimen/spacing_normal">
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_weight="1"
+                    android:layout_height="wrap_content"
+                    android:text='@string/restrict_all'/>
```

Comment:
Change to default text color and size.

Reply:
Default?

Reply:
No, I didn't mean that you define some style. I meant that keep the text color as black and the text size similar to what we keep throughout the app.

Reply:
That is what the purpose of a style is. You are asking to temporarily include the style here when it surely needs to be changed later. And I can't find the size that you want. In other places we are unevenly using different TextAppearance styles or directly used color and size, which I can't decide unless you give me a mock. I don't know which one of them you want.
Small changes like line breaks brackets and fonts are taking the focus away from important issues. With the current  scenarios in channels, I think we need to focus harder on significant issues than on appearance right now. They require switching back, squashing and violation checks to be done again. I will personally define style for each UI element and remove all the extra line breaks and if brackets from the project after gsoc.

Reply:
As per your message which i guess is deleted by you now. 
Thanks for explaining the purpose of style and the procedure of updating PR's. Since you said that you'll be handling styles in a different issue therefore, is said that just add 2 attributes so that the UI looks consistent with other screens.
I understand that these days we need to get work done asap and that's why I removed my review even before you wrote this comment so that this PR can be merged right away. 

When the app is reviewed in meeting every minute detail including the UI is observed. That's why I suggested to change it. Still, if you have any complaints from me, please post it in gitter channel.